### ISSUE TITLE
Do the image samples in the seen category in the validation set appear in the training set?

### ISSUE THREAD

**Issue by Harzva** [daoyuan98/Relation-CZSL#5]
Do the image samples in the seen category in the validation set appear in the training set?
